mobile-app/app/screens/TransactionAuthorization/PasscodePrompt.tsx

```diff
@@ -263,37 +237,15 @@ export const PasscodePrompt = React.memo((props: PasscodePromptProps): JSX.Eleme
   )
 })
 
-function SuccessMessage ({ message }: { message?: { title: string, description: string } }): JSX.Element | null {
-  const { isLight } = useThemeContext()
-  if (message === undefined) {
-    return null
-  }
-
+function SuccessIndicator (): JSX.Element {
   return (
-    <View style={tailwind('flex-col items-center p-6')}>
-      {isLight ? <TransactionSignedLight /> : <TransactionSignedDark />}
-      <ThemedText style={tailwind('text-center text-xl font-bold mt-5')}>
-        {message.title}
-      </ThemedText>
-
-      <ThemedText style={tailwind('text-sm text-center')}>
-        {message.description}
-      </ThemedText>
-    </View>
-  )
-}
-
-function Loading ({ message }: { message?: string }): JSX.Element | null {
-  if (message === undefined) {
-    return null
-  }
-  return (
-    <View style={tailwind('flex-row justify-center p-2')}>
-      <ThemedActivityIndicator />
-
-      <ThemedText style={tailwind('ml-2 text-sm')}>
-        {message}
-      </ThemedText>
+    <View style={tailwind('flex flex-col items-center py-4 my-1')}>
+      <MaterialIcons
```

Comment:
Don't have to use `<MaterialIcons/>`, we wrapped a layer on top of it with `<ThemedIcon/>`
Try looking into `mobile-app/app/components/themed/ThemedIcon.tsx` to see how it is generic class to support multiple icon set